data/ke2022daily/ke2022daily.yaml

```diff
@@ -0,0 +1,5379 @@
+# yaml-language-server: $schema=../.schema.yaml
+title: Daily Longitudinal Sampling of SARS-CoV-2 Infection Reveals Substantial Heterogeneity
+  in Infectiousness
+doi: 10.1038/s41564-022-01105-z
+description: >
+  The authors studied the dynamics of infectious virus and viral RNA shedding for
+  SARS-CoV-2 during acute infection through daily longitudinal sampling of 60 individuals
+  for up to 14 days. Nasal swab and saliva samples were collected daily and tested
+  for University of Illinois at Urbana-Champaign faculty, staff, and students (during
+  the fall of 2020 and spring of 2021) who reported a negative RT-qPCR test result
+  in the past 7 days and were either within 24 h of a positive RT-qPCR result or within
+  5 days of exposure to someone with a confirmed positive RT-qPCR result.
+analytes:
+  nasal_SARSCoV2:
+    description: >
+      SARS-CoV-2 RNA genome copy concentration in mid-turbinate nasal swab (nasopharyngeal
+      swab) samples. Note that the unit of these measurements is per mL: this is because
+      nasal swab samples were each collected in 3 mL of VTM. The calibration curve
+      for nasal samples was in the Supplemental Table S10 in Ke et al. (2022).
+    specimen: nasopharyngeal_swab
+    biomarker: SARS-CoV-2
+    gene_target: N1 and N2
+    limit_of_quantification: unknown
```

Comment:
Do they report Ct values for level of quantification/detection?

Reply:
There is a sentence in the paper "We quantified viral genome loads for 62 nasal samples with CN values ranging between 17 and 38. " I made an assumption that this is for quantification but as you see there are big gaps between this value and the minimum values observed.